.github/ISSUE_TEMPLATE/work-item-issue.md

```diff
@@ -0,0 +1,49 @@
+# Work Item Issue Template
+description:
+title: "[Work_Item]"
+labels: ["work item"]
+assignees: [shawnalpay, jpradocueva]
+## 1. **Problem Statement** *
+Describe the problem, issue, use case, or opportunity that this work item addresses.
+Include practitioner quotes illustrating real examples a) of questions being asked by practitioners and b) value unlocked by answering these questions, if available.
+- **What is the problem?**: Explain the context and why it needs resolution.
+- **Impact**: Describe how the problem affects users, systems, or the project.
+
+Your input ...
+
+## 2. **Objective** *
+State the objective of this work item. What outcome is expected?
+- **Success Criteria**: Define how success will be measured (e.g. metrics and KPIs).
+
+Your input ...
+
+## 3. **Supporting Documentation** *
+Include links to supporting documents such as:
+- Data Examples: [Link to data or relevant files; DO NOT share proprietary information]
+- Related Use Cases or Discussion Documents: [Link to discussion]
+- PRs or Other References: [Link to relevant references]
+
+Your input ...
+
+## 4. **Proposed Solution / Approach**
+Outline any proposed solutions, approaches, or potential paths forward.
+- **Initial Ideas**: Describe potential solution paths, tools, or technologies.
+- **Considerations**: Include any constraints, dependencies, or risks.
+
+Your input ...
+
+## 5. **Epic or Theme Association**
+> This section will be completed by the Maintainers.
+> - **Epic**: [Epic Name]
+> - **Theme**: [Theme Name, if applicable]
+
+## 6. **Stakeholders** *
+List the main stakeholders for this issue.
+- **Primary Stakeholders**: [Name/Role]
+- **Other Involved Parties**: [Names/Roles]
+
+---
+
+### **Template Usage Notes**:
```

Comment:
Light suggestions:
1. Would be this section on the top of the template, so someone doesn't scroll to it, then has to scroll back up to check/fix potential issues.
2. Would then add a 3rd point saying ~"If you have suggestions for additional spec expansion but can't fill out all fields, please fill out a [feedback] item instead".

Reply:
Discussed in the Oct 14 Maintainer call; will make change.

Reply:
Resolved per other commit.